sources/Engine.ts

```diff
@@ -278,7 +278,7 @@ export class Engine {
             if (transparent) {
               return fallbackDescriptor;
             } else {
-              throw new UsageError(`This project is configured to use ${result.spec.name}`);
+              throw new UsageError(`This project is configured to use ${result.spec.name} because ${result.target} defines a "packageManager" field`);
```

Comment:
```suggestion
              throw new UsageError(`This project is configured to use ${result.spec.name} because ${result.target} has a "packageManager" field`);
```

defines => has

I think this reads a little better